l2geth/core/state/statedb.go

```diff
@@ -69,6 +70,17 @@ func GetOVMBalanceKey(addr common.Address) common.Hash {
 	return common.BytesToHash(digest)
 }
 
+// after applying fee token
+func GetTonBalanceKey(addr common.Address) common.Hash {
+	// slot 0
+	position := common.Big0
+	hasher := sha3.NewLegacyKeccak256()
+	hasher.Write(common.LeftPadBytes(addr.Bytes(), 32))
+	hasher.Write(common.LeftPadBytes(position.Bytes(), 32))
+	digest := hasher.Sum(nil)
+	return common.BytesToHash(digest)
+}
```

Comment:
I think we can use like this. Because, almost ERC20 Token may be same the storage structure! 🙇🏼‍♂️
```suggestion
func GetTonBalanceKey(addr common.Address) common.Hash {
	return GetOVMBalanceKey(addr)
}
```